frontend/src/components/landing.tsx

```diff
@@ -1,17 +1,63 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Header from './header';
 import Footer from './footer';
 import ReadMore from './readmore';
+import { GoogleLogin, GoogleLogout } from 'react-google-login';
+import { useHistory } from "react-router-dom";
 import '../styles/header.css';
 
+const clientId = "241748771473-0r3v31qcthi2kj09e5qk96mhsm5omrvr.apps.googleusercontent.com";
```

Comment:
Do not have clientId exposed like this. Put this value in the React environment file that you have and access it from there. Access it in that way in all of the locations that you use it.